xml/art_raspberry-pi.xml

```diff
@@ -590,7 +494,7 @@ Password:
   <para>
   To boot the &rpi; from the prepared &microsd;, insert the card into the card
   slot, then connect a display, an Ethernet cable, a USB keyboard, and a mouse.
-  Power the &rpi;. If everything works, you should text on the screen.
+  Power the &rpi;. If everything works, you should see text on the screen.
```

Comment:
Good catch, but if you don't put this in a separate commit, it becomes difficult to backport.